src/table/settings/company_settings.ini

```diff
@@ -16,15 +16,15 @@ static const SettingTable _company_settings{
 [post-amble]
 };
 [templates]
-SDT_BOOL = SDT_BOOL(CompanySettings, $var,        $saveflags, $guiflags, $def,                        $str, $strhelp, $strval, $pre_cb, $post_cb, $from, $to, $cat, $extra, $startup),
-SDT_VAR  =  SDT_VAR(CompanySettings, $var, $type, $saveflags, $guiflags, $def, $min, $max, $interval, $str, $strhelp, $strval, $pre_cb, $post_cb, $from, $to, $cat, $extra, $startup),
+SDT_BOOL = SDT_BOOL(CompanySettings, $var,        $saveflags, $flags, $def,                        $str, $strhelp, $strval, $pre_cb, $post_cb, $from, $to, $cat, $extra, $startup),
+SDT_VAR  =  SDT_VAR(CompanySettings, $var, $type, $saveflags, $flags, $def, $min, $max, $interval, $str, $strhelp, $strval, $pre_cb, $post_cb, $from, $to, $cat, $extra, $startup),
 
 [validation]
 SDT_VAR = static_assert($max <= MAX_$type, "Maximum value for CompanySettings.$var exceeds storage size");
 
 [defaults]
 saveflags= 0
-guiflags = SF_PER_COMPANY
+flags    = SF_PER_COMPANY
```

Comment:
I think this three-way-rename "guiflags" -> "flags" -> "saveflags" is unfair for rebasing patches/forks.
How about "sf"?